studio/desk-structure.js

```diff
@@ -26,6 +38,83 @@ export default () =>
             .documentId('homepage'),
         )
         .icon(MdHome),
+      S.listItem()
+        .title('Services Page')
+        .child(
+          S.editor()
+            .id('servicesPage')
+            .schemaType('servicesPage')
+            .documentId('servicesPage'),
+        )
+        .icon(MdBuild),
+      S.divider(),
+      S.listItem()
+        .title('About Page')
+        .child(
+          S.editor()
+            .id('aboutPage')
+            .schemaType('aboutPage')
+            .documentId('aboutPage'),
+        )
+        .icon(MdBuild),
+      S.divider(),
+      S.listItem()
+        .title('Contact Page')
+        .child(
+          S.editor()
+            .id('contactPage')
+            .schemaType('contactPage')
+            .documentId('contactPage'),
+        )
+        .icon(MdBuild),
+      S.divider(),
+      S.listItem()
+        .title('Indiv Page')
+        .child(
+          S.editor()
+            .id('indivPage')
+            .schemaType('indivPage')
+            .documentId('indivPage'),
+        )
+        .icon(MdBuild),
+      S.divider(),
+      S.listItem()
+        .title('Lent Page')
+        .child(
+          S.editor()
+            .id('lentPage')
+            .schemaType('lentPage')
+            .documentId('lentPage'),
+        )
+        .icon(MdBuild),
+      S.divider(),
+      S.listItem()
+        .title('Oslo Jazz Page')
+        .child(
+          S.editor()
+            .id('osloJazzPage')
+            .schemaType('osloJazzPage')
+            .documentId('osloJazzPage'),
+        )
+        .icon(MdBuild),
+      S.divider(),
+      S.listItem()
+        .title('Tabetalt Page')
+        .child(
+          S.editor()
+            .id('tabetaltPage')
+            .schemaType('tabetaltPage')
+            .documentId('tabetaltPage'),
+        )
+        .icon(MdBuild),
```

Comment:
The pages `indiv`, `oslojazz`, `lent` and `tabetalt` are clients. I think its better to keep the references to these clients in Sanity, what do you think? Seeing as clients come and go?

Reply:
I think it is possible and will be more correct so that this data can be manually deleted and added

Reply:
What do you mean by manually deleted and added?

Reply:
I mean, save the list of clients in the Sanity store and then bind some objects/forms to them, as well as to all other pages, right?